paper-api/src/main/java/org/bukkit/inventory/view/EnchantmentView.java

```diff
@@ -36,14 +37,13 @@ public interface EnchantmentView extends InventoryView {
      *
      * @return The enchantment offers that are provided
      */
-    @NotNull
-    EnchantmentOffer[] getOffers();
+    @NotNull EnchantmentOffer @Nullable [] getOffers();
```

Comment:
Wrong way around.
The arrays are not null, its elements may be.
Same for below.